tests/es6modules/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -80,7 +80,7 @@ export default async function foo() {}
 exports[`export_default_function_declaration_async.js - flow-verify 1`] = `
 export default async function foo() {}
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-export default (async function foo() {});
+export default async function foo() {}
```

Comment:
Do we know why this happens? Is it valid?

Reply:
No idea, but it is valid flow syntax: https://flow.org/try/#0KYDwDg9gTgLgBAE2AMwIYFcA29UGcCeAdgMZzLokwCWEhZEEAFAJRwDeAvkA

Reply:
See https://github.com/prettier/prettier/issues/5084#issuecomment-420489740.